cortex-cms/cortex
# Railway-Oriented Error Handling

**Purpose**
This PR migrates from our old "exceptions as control flow" paradigm to a much more flexible, maintainable and understandable railway-oriented approach.

**JIRA**
N/A

**Changes**
* Improvements and fixes
  * Sets up our Controllers and Transactions to take advantage of a Railway-Oriented error handling flow
  * Returns to Cell-rendered Media ContentType CRUD view
  * Various refactors

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
N/A

**Feature Server**
N/A

**How to Verify These Changes**
* Specific pages to visit
  * ContentType Create and Update views

* Steps to take
  * Ensure Create/Update delete still function as expected

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies**
  * N/A

**Additional Information**
N/A
